layouts/partials/images/picture.html

```diff
@@ -1,16 +1,6 @@
 <picture{{ with .wrapperClass }} class="{{ . }}"{{ end }}>
-  {{- range .sources }}
-    <source srcset="{{ .srcset }}" type="{{ .type }}" />
-  {{- end }}
-  <img
-    class="{{ .className }}"
-    src="{{ .src }}"
-    alt="{{ .alt }}"
-    {{ if .lazyLoading }}loading="lazy"{{ end }}
-    {{ with .naturalHeight }}height="{{ . }}"{{ end }}
-    {{ with .naturalWidth }}width="{{ . }}"{{ end }}
-    {{ with .style }}{{ printf `style="%s"` (delimit . `; `) | safeHTMLAttr }}{{ end }}
-    {{ if and .original .originalSrc }}data-src="{{ .originalSrc }}"{{ end }}
-    {{ if and .original .originalWidth }}data-width="{{ .originalWidth }}"{{ end }}
-    {{ if and .original .originalHeight }}data-height="{{ .originalHeight }}"{{ end }} />
+{{- range .sources }}
+<source srcset="{{ .srcset }}" type="{{ .type }}" />
+{{- end }}
+<img class="{{ .className }}" src="{{ .src }}" alt="{{ .alt }}"{{ if .lazyLoading }} loading="lazy"{{ end }}{{ with .naturalHeight }} height="{{ . }}"{{ end }}{{ with .naturalWidth }} width="{{ . }}"{{ end }}{{ with .style }}{{ printf ` style="%s"` (delimit . `; `) | safeHTMLAttr }}{{ end }}{{ if and .original .originalSrc }} data-src="{{ .originalSrc }}"{{ end }}{{ if and .original .originalWidth }} data-width="{{ .originalWidth }}"{{ end }}{{ if and .original .originalHeight }} data-height="{{ .originalHeight }}"{{ end }} />
```

Comment:
Is this neccessary? If not, please do not inline it, it's hard to read (review) and maintain the code.
Hugo has `--minify` flag to minify the HTML codes, we don't need to take care of this manually.

Reply:
I tested with this code:
```html
<picture{{ with .wrapperClass }} class="{{ . }}"{{ end }}>
{{- range .sources }}
<source srcset="{{ .srcset }}" type="{{ .type }}" />
{{- end }}
<img class="{{ .className }}"
  src="{{ .src }}"
  alt="{{ .alt }}"
  {{ if .lazyLoading }}loading="lazy"{{ end }}
  {{ with .naturalHeight }}height="{{ . }}"{{ end }}
  {{ with .naturalWidth }}width="{{ . }}"{{ end }}
  {{ with .style }}{{ printf ` style="%s"` (delimit . `; `) | safeHTMLAttr }}{{ end }}
  {{ if and .original .originalSrc }}data-src="{{ .originalSrc }}"{{ end }}
  {{ if and .original .originalWidth }} data-width="{{ .originalWidth }}"{{ end }}
  {{ if and .original .originalHeight }} data-height="{{ .originalHeight }}"{{ end }} />
</picture>
```
In this code, I only removed HTML tag indentation.

But this will leave `/>` appear in render result.

![1](https://github.com/hugomods/images/assets/37061801/87289f1b-89d4-435f-9980-2df3f6bc6ba7)

I also removed two whitespace `  ` in front of `src=`, `alt=` etc. but that didn't make any difference.

I think every whitespace and line break must be removed to prevent this kind of problem.

Reply:
Please share your use case (how do you use it and reproduce), not sure what cause this.